src/views/createEditCommunity/__tests__/create.test.tsx

```diff
@@ -1028,7 +1054,7 @@ describe('create community', () => {
                         data: communityDummyData,
                         error: undefined,
                     });
-                }, 5000);
+                }, 1000);
```

Comment:
Any reason for this change?

Reply:
Decrease the amount of time of test wait state on this promise.